File: src/main/java/com/epicsquid/whoosh/init/WhooshLang.java
package com.epicsquid.whoosh.init;

import com.epicsquid.whoosh.Whoosh;
import com.tterrag.registrate.Registrate;
import net.minecraft.network.chat.TranslatableComponent;
import net.minecraft.resources.ResourceLocation;

public class WhooshLang {


	private static final Registrate REGISTRATE = Whoosh.registrate();

	public static final TranslatableComponent TRANSPORTER = REGISTRATE.addLang("gui", new ResourceLocation(Whoosh.MODID, "transporter"), "Transporter");
	public static final TranslatableComponent BUTTON_WHOOSH = REGISTRATE.addLang("gui", new ResourceLocation(Whoosh.MODID, "button.whoosh"), "Whoosh");
	public static final TranslatableComponent ENTER_NAME = REGISTRATE.addLang("gui", new ResourceLocation(Whoosh.MODID, "enter_name"), "Enter Name");
	public static final TranslatableComponent MODE_BLINK = REGISTRATE.addLang("info", new ResourceLocation(Whoosh.MODID, "transporter.mode.blink"), "Blink");
	public static final TranslatableComponent MODE_TELEPORT = REGISTRATE.addLang("info", new ResourceLocation(Whoosh.MODID, "transporter.mod.teleport"), "Teleport");
	public static void init() {
	}
}

File: src/main/java/com/epicsquid/whoosh/init/WhooshLang.kt
package com.epicsquid.whoosh.init

import com.tterrag.registrate.Registrate
import dev.epicsquid.whoosh.Whoosh
import net.minecraft.resources.ResourceLocation

object WhooshLang {
    private val REGISTRATE: Registrate = Whoosh.registrate()
    @JvmField
	val TRANSPORTER: TranslatableComponent =
        REGISTRATE.addLang("gui", ResourceLocation(Whoosh.MODID, "transporter"), "Transporter")
    @JvmField
	val BUTTON_WHOOSH: TranslatableComponent =
        REGISTRATE.addLang("gui", ResourceLocation(Whoosh.MODID, "button.whoosh"), "Whoosh")
    @JvmField
	val ENTER_NAME: TranslatableComponent =
        REGISTRATE.addLang("gui", ResourceLocation(Whoosh.MODID, "enter_name"), "Enter Name")
    @JvmField
	val MODE_BLINK: TranslatableComponent =
        REGISTRATE.addLang("info", ResourceLocation(Whoosh.MODID, "transporter.mode.blink"), "Blink")
    @JvmField
	val MODE_TELEPORT: TranslatableComponent =
        REGISTRATE.addLang("info", ResourceLocation(Whoosh.MODID, "transporter.mod.teleport"), "Teleport")

    fun init() {}
}
